src/sqlfluff/core/linter/linter.py

```diff
@@ -302,6 +303,35 @@ def _warn_unfixable(code: str):
             f"One fix for {code} not applied, it would re-cause the same error."
         )
 
+    @staticmethod
+    def get_line_ending_export_style(config: FluffConfig) -> Optional[str]:
+        """Get the line_ending_export_style to be used to export the fixed file.
+
+        Options are:
+        - autodetect: Use system default line ending.
+        - lf: Unix-style line ending.
+        - crlf: Windows-style line ending.
+        """
```

Comment:
You might want to add in Mac class style endings, which are just cr

Reply:
@WittierDinosaur good point, it's just `cr: \r` so easy to add 👍